cypress/integration/articleFlows/commentOnArticle.spec.js

```diff
@@ -150,20 +156,20 @@ describe('Comment on articles', () => {
         getCommentCombobox();
         getCommentCombobox().should('have.focus');
 
-        getCommentCombobox().type('ser');
+        getCommentCombobox().type('s');
       });
 
       cy.findByText('No results found').should('exist');
       getCommentCombobox().type(' ');
 
       cy.findByText('No results found').should('not.exist');
       getCommentPlainTextBox().should('have.focus');
-      getCommentPlainTextBox().should('have.value', 'Some text @user ');
+      getCommentPlainTextBox().should('have.value', 'Some text @us ');
```

Comment:
Question (non-blocking): Curious why we only search for 2 characters now.

Reply:
Just related to the comment above ^ - it makes more sense than mocking out every single search call up to the full word (if we stuck with `user` here we'd need to mock `us`, `use`, `user`